README.md

```diff
@@ -18,7 +18,84 @@ npm install geckoboard
 
 ## Use
 
-The latest documentation and user guide can be found on the Geckoboard developer docs (https://developer.geckoboard.com/)
+The latest documentation and user guide can be found on the Geckoboard developer docs (https://developer.geckoboard.com/) 
+
+### Ping to authenticate
```

Comment:
This isn't a necessary step, perhaps we don't need to document it?

Reply:
Yes,  I'm happy to remove or clarify it means `Ping to check you're authenticated`